meteor/client/layout.js

```diff
@@ -44,6 +44,15 @@ if (Meteor.isClient) {
             Session.set('ipintelipaddress',($(e.target).attr('data-ipaddress')));
             $('#modalintelwindow').modal()
         },
+        "click .ipmenu-search": function(e,t){
+            var ipText=$(e.target).attr('data-ipaddress');
+            console.log("IP: " + ipText)
+            var searchDomain=getSetting('kibanaURL');
+            var searchPath="#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-1h,mode:quick,to:now))&_a=(columns:!(_source),index:AWdcCjsPo9lh6BmjcvcD,interval:auto,query:(query_string:(analyze_wildcard:!t,query:'details.sourceipaddress:"+ipText+"')),sort:!(utctimestamp,desc))";
```

Comment:
This does not work as the index hash "AWdcCjsPo9lh6BmjcvcD" is specific to each kibana installation. (meaning this won't always be the hash of the index pattern name)

Reply:
good point.

Reply:
I've replaced with the plaintext version of "events-weekly"
Also asked you to automagically populate the id of index-patterns so we don't have to deal with hashes, this should resolve the issue :) 

Thanks!